Client/mods/deathmatch/logic/luadefs/CLuaCompatibilityDefs.cpp

```diff
@@ -71,6 +71,8 @@ void CLuaCompatibilityDefs::LoadFunctions()
         {"getControlState", CLuaPedDefs::GetPedControlState},
         {"setCameraShakeLevel", ArgumentParserWarn<false, CLuaCameraDefs::SetCameraDrunkLevel>},
         {"getCameraShakeLevel", ArgumentParserWarn<false, CLuaCameraDefs::GetCameraDrunkLevel>},
+        {"base64Encode", CLuaCryptDefs::EncodeString},
+        {"base64Decode", CLuaCryptDefs::DecodeString},
```

Comment:
What about `{"base64Encode", ArgumentParserWarn<false, Base64encode>},` in CLuaCryptDefs.cpp?

Reply:
I deleted them from here because we talked on Discord about keeping the functions forever.